src/components/visualizations/chart/highcharts/test/getZeroAlignConfiguration.spec.ts

```diff
@@ -205,6 +209,171 @@ describe("getZeroAlignConfiguration", () => {
         });
     });
 
+    describe("getMinMaxInfo with some null data", () => {
+        const SERIES: ISeriesItem[] = [
+            {
+                yAxis: 0,
+                data: [null, null].map((num: number) => ({ y: num })),
```

Comment:
Could we re-use this function?
and change `num: number` => `num: number | null`

Reply:
done